FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice_utest.c

```diff
@@ -631,227 +645,613 @@ void test_task_delay_until_with_config_assert( void )
 }
 
 /**
- * @brief prvAddNewTaskToReadyList - add a newly created task to the list of ready tasks
- *
- * This test creates two tasks, the second after suspending the first and then
- * starts the scheduler.
+ * @brief xTaskPriorityDisinherit - disinherit priority, potentially restoring the original priority used before escalation due to mutex usage
  *
  * <b>Coverage</b>
  * @code{c}
- * if( uxCurrentNumberOfTasks == ( UBaseType_t ) 1 )
+ *     if( pxTCB->uxPriority != pxTCB->uxBasePriority )
+ *     {
+ *     ...
+ *          if( pxTCB->uxMutexesHeld == ( UBaseType_t ) 0 )
  * @endcode
- * As two tasks arecreated, this covers both branches of the above conditional
- * in addition to the function body.
+ *
+ * Cover the case where priority has changed due to inheritance
+ * because of a shared mutex.
+ *
  */
-void test_coverage_prvAddNewTaskToReadyList_create_two_tasks_with_the_first_suspended( void )
```

Comment:
It looks like some of the test cases are removed in this PR?
test_coverage_prvAddNewTaskToReadyList_create_two_tasks_with_the_first_suspended
test_coverage_prvAddNewTaskToReadyList_create_more_idle_tasks_than_cores
test_coverage_vTaskCoreAffinitySet_task_core_affinity_set_task_implied
test_coverage_vTaskCoreAffinitySet_task_core_affinity_set_task_explicit
....
Why are we removing existing test cases?